packages.nix

```diff
@@ -6,6 +6,7 @@ with haskellPackages;
   ##############################################################################
   # Add general packages here                                                  #
   ##############################################################################
+  cabal-macosx
```

Comment:
This might make sense to put in [generalDevTools](https://github.com/reflex-frp/reflex-platform/blob/develop/default.nix#L441)